rllib/algorithms/ppo/ppo.py

```diff
@@ -450,17 +451,19 @@ def training_step(self) -> ResultDict:
             # New Episode-returning EnvRunner API.
             else:
                 if self.workers.num_remote_workers() <= 0:
-                    episodes = [self.workers.local_worker().sample()]
+                    episodes: List[SingleAgentEpisode] = [
```

Comment:
Nice! Thanks for adding the typehints.

Reply:
I thought this is better readable for users, to see from whereon we switch to `SampleBatch`.